style.css

```diff
@@ -0,0 +1,126 @@
+body {
+    margin: 0;
+    color: white;
+    font-size: 14px;
+    font-family: Montserrat;
+}
+
+ul {
+    padding: 0;
+    list-style: none;
+    margin: 0;
+}
+
+.top-banner {
+    background-image: url("img/cover.png");
+    background-repeat: no-repeat;
```

Comment:
I would add background-size:cover here so that the background image stretches across the screen

Reply:
And instead of cover.png you can download background picture already with filter applied. Otherwise, everything looks good as per my not prof opinion:)

Reply:
Thank you! I will do it in the next Home work!